src/Integration/Helpers/IKnownUIContexts.cs

```diff
@@ -23,11 +23,19 @@
 
 namespace SonarLint.VisualStudio.Integration
 {
+    public interface IUIContext
+    {
+        bool IsActive { get; }
+    }
+
     public interface IKnownUIContexts
     {
         event EventHandler<UIContextChangedEventArgs> SolutionBuildingContextChanged;
         event EventHandler<UIContextChangedEventArgs> SolutionExistsAndFullyLoadedContextChanged;
         event EventHandler<UIContextChangedEventArgs> CSharpProjectContextChanged;
         event EventHandler<UIContextChangedEventArgs> VBProjectContextChanged;
+
+        IUIContext SolutionExistsAndFullyLoadedContext { get; }
+        IUIContext SolutionExistsAndNotBuildingAndNotDebuggingContext { get; }
```

Comment:
Previously the wrapper just handled events. Now it needs to return UI contexts that can be queried.